mail_forward/wizard/mail_forward.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+# © 2014-2015 Grupo ESOC <www.grupoesoc.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class MailForwardComposeMessage(models.TransientModel):
+    """Allow forwarding a message.
+
+    It duplicates the message and optionally attaches it to another object
+    of the database and sends it to another recipients than the original one.
+    """
+
+    _name = "mail_forward.compose.message"
+    _inherits = {"mail.compose.message": "original_wizard_id"}
+
+    _models = [
```

Comment:
What does it do? Actually I was just adding a boolean to `ir.model` called `mail_forward_target`. Seems the easiest way.